src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -270,25 +267,35 @@ private boolean isBlacklistProject(String project) {
 		return ArrayUtils.contains(operationsProjects, project.toLowerCase());
 	}
 
-	private boolean isClusterAdmin(final String token){
-		ConfigBuilder builder = new ConfigBuilder()
-				.withOauthToken(token);
+	private boolean isOperationsUser(final String username){
+		ConfigBuilder builder = new ConfigBuilder();
+		OpenShiftClient osClient = new DefaultOpenShiftClient(builder.build());
```

Comment:
I think this class is 'Closeable' which means you can:
```
try(OpenShiftClient osClient = new DefaultOpenShiftClient(builder.build())){
}catch(Exception){}
```
and not even need to explicit close the client